### PR TITLE
LPS-115061 v2

### PR DESCRIPTION
**Pull request by MarcellGyopos**
Hi @natocesarrego ,
Could you please review this pull request?
Thank you and best regards,
Marci

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValuesValidatorImpl.java [hunk 1]
```diff
@@ -225,22 +225,23 @@ protected boolean isNull(
...
 
-				return true;
+					return false;
+				}
 			}
-		}
 
-		return false;
+			return true;
+
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValuesValidatorImpl.java]: @MarcellGyopos, why did you decide to change the current logic? Check for `value.getAvailableLocales().isEmpty()` on the previous `if` isn't enough?

Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValuesValidatorImpl.java]: Oh... After some re-reading, I believe that I got your point. 😁 I'll forward your pull to the DDM/DDL official reviewer, once he already back from PTO. 🙂

**Comment by natocesarrego** [comment]
Pull request forwarded to https://github.com/leoadb/liferay-portal/pull/917. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/799820444756~3...799820444756) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValuesValidatorImpl.java (hunk 1 above)
Comment (MarcellGyopos) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValuesValidatorImpl.java]: Thank you @natocesarrego :)

**Pull request closed without merging**